lib/ui/fixtures/ui_test.dart

```diff
@@ -450,6 +450,13 @@ void hooksTests() {
     }
   });
 
+  test('deprecated region equals', () {
+    const Locale x = Locale('en', 'ZR');
```

Comment:
A comment explaining why these are equal would be welcome, e.g. `// ZR (Zaire) is a deprecated country code for  Congo (CD), and so create equal Locale objects` or something.

Reply:
done